apps/frontend/src/dashboard/Dashboard.tsx

```diff
@@ -28,6 +28,10 @@ const Dashboard: React.FC<{}> = () => {
   if (dashboard) {
     const data = {
       ...dashboard,
+      newsAndEvents: dashboard.newsAndEvents.map((n) => ({
+        ...n,
+        href: `/news-and-events/${n.id}`,
```

Comment:
We're trying to migrate to central route path definitions, could you use `NEWS_AND_EVENTS_PATH` anywhere you touch to avoid introducing further duplicates?